jujugui/static/gui/src/app/views/topology/service.js

```diff
@@ -151,10 +152,32 @@ YUI.add('juju-topology-service', function(Y) {
       } else if (d.highlighted) {
         rerenderRelations = true;
       } else {
-        curr_node.attr({
-          'stroke': '#888888',
-          'stroke-width': 1
+        var units = d.units._items;
+        units.forEach(function(unit) {
```

Comment:
@hatched noted this would result in a performance hit in large bundles. @makyo do you think it would be possible to use the unitServiceList to determine state?